extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -237,6 +237,7 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
 
       ctrl.postSvgToServer = function(dimensions, resampledFile) {
+        ctrl.isLoadingIndicatorShown = true;
```

Comment:
Please change this to `loadingIndicatorIsShown` instead.
Your version would be more appropriate for a method name.
See [this](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript) for reference.

Reply:
Done